NTumbleBit/ClassicTumbler/Client/PaymentStateMachine.cs

```diff
@@ -459,27 +464,27 @@ public CycleProgressInfo Update()
 								var transaction = PromiseClientSession.GetSignedTransaction(tumblingSolution);
 								Logs.Client.LogDebug("Got puzzle solution cooperatively from the tumbler");
 
-							    Console.WriteLine("TumblerCashOut hex: " + transaction.ToHex());
+								Console.WriteLine("TumblerCashOut hex: " + transaction.ToHex());
```

Comment:
These console outputs should probably be changed to use the logger, I think they were committed by accident when I was diagnosing an error some time ago

Reply:
Changed to LogDebug